ramalama/benchmarks/utilities.py

```
@@ -79,10 +81,14 @@ def print_bench_results(records: list[BenchmarkRecord]):
     optional_fields = ["id", "engine", "date"]
     for field in optional_fields:
         if all(not row.get(field) for row in rows):
-            for row in rows:
-                row.pop(field, None)
+            rows[0].pop(key, None)
```

Ruff (0.15.15)

[error] Undefined name `key`

(F821)